src/index.html

```diff
@@ -11,7 +11,219 @@
+            E-mail:
+            <input
+              type="email"
+              name="email"
+              placeholder="email@example.com"
+              id="email"
+              required
+              minlength="1"
+            >
+          </label>
+        </div>
+
+        <div class="labels">
+          <label for="password">
+            Password:
+            <input
+              type="password"
+              name="password"
+              id="password"
+              required
+              minlength="5"
+              maxlength="12"
+            >
+          </label>
+        </div>
+      </fieldset>
+
+      <fieldset class="box">
+        <legend>An interesting fact about you!</legend>
+
+        <div class="labels">
+          <span class="radio">Do you love cats?</span>
+          <label class="radio" for="loveCatsYes">
+            Yes
+            <input
+              type="radio"
+              name="cats"
+              id="loveCatsYes"
+              value="Yes"
+            >
+          </label>
+
+          <label class="radio" for="loveCatsNo">
+            No
+            <input
+              type="radio"
+              name="cats"
+              id="loveCatsNo"
+              value="No"
+            >
+          </label>
+        </div>
+
+        <div class="labels">
+          <label for="color">
+            What is your favorite color?
+            <input
+              type="color"
+              name="color"
+              id="color"
+            >
+          </label>
+        </div>
+
+        <div class="labels">
+          <label for="goToBed">
+            What time do you go to bed?
+            <input
+              type="time"
+              name="goToBed"
+              id="goToBed"
+            >
+          </label>
+        </div>
+
+        <div class="labels">
+          <label for="cars">
+            What are your favorite brand of cars?
+            <select
+              name="cars"
+              id="cars"
+              multiple
+            >
+              <option value="BMW">BMW</option>
+              <option value="Audi">Audi</option>
+              <option value="Lada">Lada</option>
+              <option disabled selected></option>
+            </select>
+          </label>
+        </div>
+
+        <div class="labels">
+          <label for="rateWork">
+            How do you rate our work?
+            <input
+              type="range"
+              id="rateWork"
+              name="rateWork"
+            >
+          </label>
+        </div>
+      </fieldset>
+
+      <fieldset class="box">
+        <legend>Additional info:</legend>
+
+        <div class="labels">
+          <label for="comments">
+            Comments:
+            <textarea
+              name="comments"
+              id="comments"
+              maxlength="500"
+            >
+            </textarea>
+          </label>
+        </div>
+
+        <div class="labels">
+          <label for="recommnded">
```

Comment:
https://mateacademy.slack.com/archives/C02LA8ED589/p1637055430264700

```suggestion
          <label for="recommended">
```